app/src/main/res/values/strings.xml

```diff
@@ -407,6 +407,7 @@ How to translate with transifex:
     <string name="nc_sent_poll" formatted="true">%1$s sent a poll.</string>
     <string name="nc_sent_location" formatted="true">%1$s sent a location.</string>
     <string name="nc_sent_voice" formatted="true">%1$s sent a voice message.</string>
+    <string name="nc_sent_deck_card" formatted="true">%1$s: %2$s</string>
```

Comment:
%1$s sent a deck card.

Reply:
The string is modified to align with web

Reply:
Same as above